java/src/jmri/jmrix/loconet/uhlenbrock/UhlenbrockPacketizer.java

```diff
@@ -76,13 +75,10 @@ public void sendLocoNetMessage(LocoNetMessage m) {
         if (log.isDebugEnabled()) {
             log.debug("queue LocoNet packet: {}", m.toString());
         }
-        // in an atomic operation, queue the request and wake the xmit thread
+        // queue the request
         try {
-            synchronized (xmtHandler) {
-                xmtLocoNetList.addLast(m);
-                xmtList.addLast(msg);
-                xmtHandler.notify();
-            }
+            xmtLocoNetList.addLast(m);
+            xmtList.add(msg);
```

Comment:
Is xmtLocoNetList synchronized? Or doesn't it matter?

Reply:
Thanks for the careful check.  Just to be on the safe side, I've changed it to a `ConcurrentLinkedQueue`.  Thanks again.